proxy/proxy_script.js

```diff
@@ -0,0 +1,18 @@
+// Keep track of any local settings for the addon.
+let settings = {
+  enabled: false,
+};
+
+// Required PAC function that will be called to determine
+// if a proxy should be used.
+function FindProxyForURL(url, host) {
+  if (settings.enabled) {
+    return "DIRECT 1.2.3.4:8080"; // Update to return proxy server rules (e.g. "PROXY 1.2.3.4:8080")
+  }
+  return "DIRECT 1.2.3.4:8080";
```

Comment:
Nothing will happen. It's hard to really have anything useful without setting up a proxy server. 

It should really just be returning "DIRECT" here, not "DIRECT 1.2.3.4:8080", but I discovered a bug where "DIRECT" requires an argument to be passed for it to work - I've filed bug 1355198 to fix this.